calico_rkt/calico_rkt.py

```diff
@@ -229,21 +238,52 @@ def _apply_rules(profile_name, client):
     try:
         profile = client.get_profile(profile_name)
     except:
-        print_stderr("Error: Could not apply rules. Profile not found: %s, exiting" % profile_name)
+        logger.info("ERROR: Could not apply rules. Profile not found: %s, exiting" % profile_name)
         sys.exit(1)
 
     profile.rules = _create_rules(profile_name)
     client.profile_update_rules(profile)
-    print_stderr("Finished applying rules.")
+    logger.info("Finished applying rules.")
+
 
-def _allocate_IP():
+def _generate_pool(client):
     """
-    Determine next available IP for pool in input_ and assign it
+    Take Input subnet (global), create IP pool in datastore
+    Will complete silently if it exists
+    return IPPool  object of subnet pool
     """
-    pool = input_json['ipam']['subnet']
-    candidate = SequentialAssignment().allocate(IPNetwork(pool))
-    print_stderr("Using IP %s" % candidate)
-    return candidate
+    try:
+        subnet = INPUT_JSON['ipam']['subnet']
```

Comment:
I'm not a big fan of pulling the global variable into this function. It'd be much cleaner and more easily testable if you pass it in as a parameter. You can still pass in the global variable - just do it where this is called. @caseydavenport agreed?